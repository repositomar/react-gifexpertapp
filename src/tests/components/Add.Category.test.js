import React from 'react';
import {shallow} from 'enzyme';
import { AddCategory } from '../../components/AddCategory';

describe("Pruebas en componente", () => {

    const setCategories = () => {};
    
    test("Debe mostrar el componente correctamente", () => {

        const wrapper = shallow(<AddCategory setCategories={setCategories}/>);
        //console.log(wrapper)
        expect(wrapper).toMatchSnapshot();

        
    });
    
});
