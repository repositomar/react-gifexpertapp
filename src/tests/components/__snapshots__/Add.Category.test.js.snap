// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente Debe mostrar el componente correctamente 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    value=""
  />
</form>
`;
